modules/passkey/src/utils/webauthn.ts

```diff
@@ -0,0 +1,185 @@
+import { ethers } from 'ethers'
+import type { BigNumberish, BytesLike } from 'ethers'
+import CBOR from 'cbor'
+
+export function userVerificationFlag(userVerification: UserVerificationRequirement = 'preferred'): number {
+  switch (userVerification) {
+    case 'preferred':
+      return 0x01
+    case 'required':
+      return 0x04
+    default:
+      throw new Error(`user verification requirement ${userVerification} not supported`)
+  }
+}
+
+export function b2ab(buf: Uint8Array): ArrayBuffer {
+  return buf.buffer.slice(buf.byteOffset, buf.byteOffset + buf.byteLength)
+}
```

Comment:
This is only used internally in the WebAuthn shim, so I would move it there.

Reply:
Done 👍🏾